libsql-server/src/http/admin/mod.rs

```diff
@@ -429,3 +433,32 @@ async fn handle_delete_namespace<C>(
         .await?;
     Ok(())
 }
+
+#[derive(Deserialize, Default)]
+struct CheckpointReq {
+    #[serde(default)]
+    pub force: bool,
+}
+
+async fn handle_checkpoint<C>(
+    State(app_state): State<Arc<AppState<C>>>,
+    Path(namespace): Path<String>,
```

Comment:
there is an extractor for namespace name :)
```suggestion
    Path(namespace): Path<NamespaceName>,
```